js/index.js

```diff
@@ -1 +1,37 @@
 // Your code goes here
+
+
+const mouseEvent = document.querySelector("nav");
```

Comment:
glad that you found that there was a way to do this without using `querySelectorAll` and avoiding the `forEach`, I did it the same way back in the day :P